app/components/ui/atoms/PersonPod.md

```diff
@@ -63,6 +63,20 @@ Selected icon:
 />
 ```
 
+Disabled icon:
```

Comment:
Could you remove this section: https://github.com/elifesciences/elife-xpub/pull/390/files#diff-9002cb506702fd5d27099698b6102d2bR80 or add the new `isIconClickable` prop to that?